Makefile.docker

```diff
@@ -5,35 +5,35 @@ SHELL := /bin/bash
 .SHELLFLAGS = -o pipefail -euc
 .DEFAULT_GOAL := build
 
-.PHONY: all build check test cover
+.PHONY: all build check test cover update-deps restore
 
-CONSUL_REF := 158eabdd6f2408067c1d7656fa10e49434f96480
-ETCD_REF := release-2.2
-
-ROOT := /go
-CDWD := cd ${ROOT}/src/containerbuddy
+PACKAGE := github.com/joyent/containerbuddy
+SRC  := /go/src/${PACKAGE}
+CDWD := cd ${SRC}
+NO_VENDOR:= $(shell cd ${SRC} && go list ./... | grep -v /vendor/)
 
 all: build cover
 
-build: /go/src/github.com
-	${CDWD} && go build -a -o /build/containerbuddy -ldflags "$(LDFLAGS)"
+restore:
```

Comment:
I understand that the action here is to "restore" but maybe this should be called `vendor` so that we're taking advantage of make and not re-creating the dependencies?